### DISCUSSION TITLE
Need a new logo for lsd

### DISCUSSION THREAD

**Discussion by zwpaper** [lsd-rs/lsd#824] [General]
Thanks so much to @Peltoche, now we host https://github.com/lsd-rs/lsd under https://github.com/lsd-rs, and we can try to build a more active org.

with the brand-new start, we may need a new logo for lsd.

I am proposing one, but my picturing skill is inferior, so I would describe it here:

1. the logo mainly comes from Rust
2. replacing the `R` inside the Rust logo with `ls` or `files icon`
3. change the outside circle to a format like a `D`

ref, Rust logo:
<img src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/rust/rust-plain.svg" width="200"/>

**Comment by zwpaper** [comment]
ChatGPT has created one and looks not that bad, let's give it a try and we can adjust it if anyone better 

let's make this one alpha1!

![lsd-logo-alpha](https://github.com/lsd-rs/lsd/assets/3764335/3498ffb7-5c8a-41ea-a7a3-a8aeb92b40ef)

**Comment by danie-dejager** [comment]
what about these two?
![image](https://github.com/user-attachments/assets/0ea6bbb1-ccba-4198-90de-70bc9ac59e41)
